### PR TITLE
Update tags and fix issues of clean-install tests

### PR DESCRIPTION
**Pull request by barbacenasmc**
**List of changes:**

1. added common step def for importing germplasm list and used them in clean-install scenarios that require germplasm list before running the test
2. updated selectProgram function in selecting default program
3. updated navigation.ts
-- added default condition for "default dashboard"
-- used different verification steps on default dashboard (checking of dropdown links only)
-- moved usage of closeReleaseNotePopupIfShown to navigation step def level instead of using it in each method in page object files
4. updated feature files to separate sanity and smoke test scenarios
5. used different import germplasm file for ungroup germplasm scenario

### CODE REVIEW

File: cypress/integration/tests/manage-germplasm/import-germplasm.feature [hunk 1]
```diff
@@ -7,14 +7,24 @@ Feature: Import Germplasm
 
 @TestCaseKey=IBP-T3177    
 @sanity-test
+@focus @clean-install
```
Comment (darla-leafnode) [inline cypress/integration/tests/manage-germplasm/import-germplasm.feature:10]: Remove @focus tag

File: cypress/integration/tests/manage-germplasm/germplasm-details.feature [hunk 2]
```diff
@@ -7,16 +7,18 @@ Feature: Germplasm Details
 
 @TestCaseKey=IBP-T3178
 @sanity-test
-    Scenario: Check pedigree tree
-        Given I am on the Germplasm Manager page
+    Scenario: Check pedigree
```
Comment (darla-leafnode) [inline cypress/integration/tests/manage-germplasm/germplasm-details.feature:10]: As discussed, create scenarios for clean and update install as for update install sanity test, we shouldn't be adding data

**Review by darla-leafnode — CHANGES_REQUESTED** [review]

File: cypress/integration/common/common-germplasm-import-step-defs.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+const importGermplasmPage = new ImportGermplasmPage()
+const importListPage = new GermplasmListImportPage()
+var listName: string;
+listName = 'list name ' + randomString();
+When('germplasm records already exists', () => {
+    // Reuse created RCBD study within session
+       if (Cypress.env('germplasmList')) {
+        return;
+    // Otherwise, import germplasm
+    } else {
```
[the remark this answers is not in the document]
Comment (barbacenasmc) [reply cypress/integration/common/common-germplasm-import-step-defs.ts:24]: Done

File: cypress/integration/common/common-germplasm-import-step-defs.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import SidebarSection, { SidebarTool } from '../pageobjects/sidebar-section'
+import { randomString } from '../../support/commands';
+
+const loginPage = new LoginPage()
+const dashboardPage = new DashboardPage()
+const sidebarSection = new SidebarSection()
+
+const manageGermplasmPage = new ManageGermplasmPage()
+const importGermplasmPage = new ImportGermplasmPage()
+const importListPage = new GermplasmListImportPage()
```
[the remark this answers is not in the document]
Comment (barbacenasmc) [reply cypress/integration/common/common-germplasm-import-step-defs.ts:16]: Done

File: cypress/integration/tests/manage-germplasm/import-germplasm.feature (hunk 1 above)
Comment (barbacenasmc) [reply cypress/integration/tests/manage-germplasm/import-germplasm.feature:10]: Done

File: cypress/integration/tests/manage-germplasm/germplasm-details.feature (hunk 2 above)
Comment (barbacenasmc) [reply cypress/integration/tests/manage-germplasm/germplasm-details.feature:10]: Done

File: cypress/integration/common/common-germplasm-import-step-defs.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import { When, And, Then } from 'cypress-cucumber-preprocessor/steps';
+import ManageGermplasmPage from '../pageobjects/germplasm/manage-germplasm-page'
+import ImportGermplasmPage from '../pageobjects/germplasm/import-germplasm-page'
+import GermplasmListImportPage from '../pageobjects/germplasm-lists/germplasm-list-import-page'
+import LoginPage from '../pageobjects/account-management/login-page'
```
Comment (darla-leafnode) [inline cypress/integration/common/common-germplasm-import-step-defs.ts:5]: Remove these unused imports:
```

import GermplasmListImportPage from '../pageobjects/germplasm-lists/germplasm-list-import-page'
import LoginPage from '../pageobjects/account-management/login-page'
```

File: cypress/integration/tests/manage-germplasm/germplasm-details.feature [hunk 6]
```diff
@@ -7,16 +7,38 @@ Feature: Germplasm Details
...
+    Scenario: Check pedigree 
+        Given I am on the Manage Germplasm page of specified program
+        When I click on the GID of a germplasm
+        And I navigate to the Pedigree tab of germplasm details
+        Then I should be able to see the pedigree tree        
+
+@TestCaseKey=IBP-T3180
+@sanity-test
+@clean-install
+    Scenario: Check pedigree
```
Comment (darla-leafnode) [inline cypress/integration/tests/manage-germplasm/germplasm-details.feature:29]: The last two scenarios should be "Check Pedigree Graph"

File: cypress/integration/common/common-germplasm-import-step-defs.ts (hunk 5 above)
Comment (barbacenasmc) [reply cypress/integration/common/common-germplasm-import-step-defs.ts]: Done

File: cypress/integration/tests/manage-germplasm/germplasm-details.feature (hunk 6 above)
Comment (barbacenasmc) [reply cypress/integration/tests/manage-germplasm/germplasm-details.feature]: Done

**Review by darla-leafnode — APPROVED** [review]

**Pull request merged**